testing/BUILD.gn

```diff
@@ -77,4 +80,55 @@ if (current_toolchain == host_toolchain) {
       "//third_party/swiftshader_flutter:swiftshader",
     ]
   }
+
+  # All targets on all platforms should be able to the Metal utilities. On
```

Comment:
typo: "to the Metal utilities" -> "to use the Metal utilities"

Reply:
Done.